panels/info/info.ui

```diff
@@ -378,65 +375,65 @@
                                         <property name="position">0</property>
                                       </packing>
                                     </child>
-				    <child>
-				      <object class="GtkBox" id="os_updates_box">
-					<property name="visible">True</property>
-					<property name="can_focus">False</property>
-					<property name="orientation">horizontal</property>
-					<property name="spacing">10</property>
-					<child>
-					  <object class="GtkLabel" id="os_updates_separator">
-					    <property name="visible">True</property>
-					    <property name="can_focus">False</property>
-					    <property name="xalign">0</property>
-					    <property name="label">—</property>
-					  </object>
-					  <packing>
-					    <property name="expand">False</property>
-					    <property name="fill">False</property>
-					    <property name="position">0</property>
-					  </packing>
-					</child>
-					<child>
-					  <object class="GtkSpinner" id="os_updates_spinner">
-					    <property name="visible">False</property>
-					    <property name="can_focus">False</property>
-					  </object>
-					  <packing>
-					    <property name="expand">False</property>
-					    <property name="fill">False</property>
-					    <property name="position">1</property>
-					  </packing>
-					</child>
-					<child>
-					  <object class="GtkLabel" id="os_updates_label">
-					    <property name="visible">True</property>
-					    <property name="can_focus">False</property>
-					    <property name="xalign">0</property>
-					    <property name="use_markup">True</property>
-					    <property name="label"></property>
-					  </object>
-					  <packing>
-					    <property name="expand">False</property>
-					    <property name="fill">False</property>
-					    <property name="position">2</property>
-					  </packing>
-					</child>
-				      </object>
-				      <packing>
-					<property name="expand">False</property>
-					<property name="fill">False</property>
-					<property name="position">1</property>
-				      </packing>
-				    </child>
-				  </object>
-				  <packing>
-				    <property name="left_attach">1</property>
-				    <property name="right_attach">2</property>
-				    <property name="top_attach">1</property>
-				    <property name="bottom_attach">2</property>
-				  </packing>
-				</child>
+            <child>
```

Comment:
This looks like it's manually edited and lost its original indentation. It would be nice to only see the real diff here.